docs/releases/versions/version-v12.0.0.md

```diff
@@ -0,0 +1,117 @@
+
+Records can also be deleted (with a grace period for appeal or undoing) which empowers administrators to enforce institutional policies and fight spam.
+
+Bulk versions of these are in the works.
+
+### Even better previewers
+TODO
+
+### GitHub integration
+TODO
+
+
+### Changes from v12.0.0b1 are officialized
+
+[An interim beta release of v12](../temporary-versions/version-v12.0.0b1.md) was made some months ago and tentatively brought a number of changes. We are happy to report all those changes are present and solidified in v12.0. Of note, those include:
+
+- Usage statistics compliant with the [MakeDataCount](https://makedatacount.org/) and [COUNTER](https://www.projectcounter.org/) standards
+- Option to skip the review process for curators and higher roles of a community
+- Record submission and inclusion in multiple communities
+- Display and dynamic administration of site banners
+- Creation of static pages via the administration panel
+- Instance-level configuration to enable metadata-only records and restricted files.
+- [New export formats](../../reference/export_formats.md) such as DACT-AP and GeoJSON
+- Contributed custom fields that are easy [to enable]((../../reference/metadata/optional_metadata.md))
+
+### Miscellaneous additions
+
+- Introduction of `PAGES_ALLOWED_EXTRA_HTML_ATTRS` and `PAGES_ALLOWED_EXTRA_HTML_TAGS` to allow specific HTML attributes and tags in Pages from [invenio-pages](PAGES_ALLOWED_EXTRA_HTML_TAGS)
+- Improvements to the `invenio rdm rebuild-all-indices` command
+- Mathematical formulas in a deposit can be rendered on the landing page via `THEME_MATHJAX_CDN = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"` and updated content-secuirty policy headers: add `cdnjs.cloudflare.com` to  `APP_DEFAULT_SECURE_HEADERS`.
+- Possibility to [add entry to a controlled vocabulary](../../customize/vocabularies/index.md#addupdate-fixtures-command) via the CLI: `invenio rdm-records add-to-fixture`
+
+## Breaking changes
+
+- See the [Requirements section](#requirements) below for what underlying requirements are now necessary
+- The [react-invenio-deposit module](https://github.com/inveniosoftware/react-invenio-deposit) has been moved to [invenio-rdm-records](https://github.com/inveniosoftware/invenio-rdm-records)
+    - Any imports should be replaced: `import { ... } from "react-invenio-deposit"` -> `import { ... } from "@js/invenio-rdm-records"`
+- Some `Overridable` React component ids may have changed. Verify that your overridden components load
+- `RoleNeed` uses the role's `id` instead of the role's `name`
+- The `/access/users` API URL has been renamed to `/access/grants` to manage access grants
+
+## Limitations and known issues
+TODO
+
+## Requirements
+
+InvenioRDM v12 now supports:
+
+- Python 3.9, 3.11 and 3.12
+- Node.js 18+
+- PostgreSQL 10+
```

Comment:
should we say at least PostgreSQL 12? since PostgreSQL 12 end of life is in four months! [link](https://endoflife.date/postgresql)